### PR TITLE
Feature/199 navbar

### PR DESCRIPTION
**Pull request by loreina**
### Tickets:

- HCK-145
- HCK-199

- Closes #740 

### List of changes:

- Remove sidebar
- Fix spacing on account and application containers
- Add page navigation for navbar (only home, profile, application + only for pre-acceptance (so hacker status = none or applied)
- Add mobile menu with same thing ^

Login page:
image

Hacker dashboard:
image

Closed burger:
image

Open burger:
image

### Type of change:

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [x] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### CODE REVIEW

**Comment by isabelledelmundo** [comment]
Just loads weird on mobile when you go to a new page but not a big deal for now
7fbd32050389eff01ad652c9a5391dd5

File: src/features/Application/ManageApplicationContainer.tsx [hunk 1]
```diff
@@ -149,53 +149,53 @@ class ManageApplicationContainer extends React.Component<
...
+          /> */}
+        <Helmet>
+          <title>
+            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'}
+            Application | McHacks 6
+          </title>
+        </Helmet>
+        <MaxWidthBox maxWidth={'500px'} m={'auto'}>
+          <H1
+            color={'#F2463A'}
```
Comment (krubenok) [inline src/features/Application/ManageApplicationContainer.tsx]: Ideally stick to using theme colour abstractions with `theme.colours.red` along with `import theme from '../../shared/Styles/theme';` somewhere in the header.

File: src/shared/Styles/GlobalStyles.tsx [hunk 2]
```diff
@@ -49,6 +49,12 @@ export const GlobalStyles = createGlobalStyle`
   .toast-notification {
     z-index: 100000;
   }
+
+  @media screen and (min-width: 768px) {
```
Comment (krubenok) [inline src/shared/Styles/GlobalStyles.tsx]: Should `@media screen and (min-width: ${(props) => props.theme.screens.smUp}) {` be used here to take advantage of the items you made in the theme file?

File: src/features/Nav/Navbar.tsx [hunk 3]
```diff
@@ -1,37 +1,102 @@
...
+        <IconContainer>
+          <a href={routes.HOME_PAGE}>
+            <Icon src={Martlet} />
+          </a>
+        </IconContainer>
+        <Links>
+          {NavItems()}
+          {CTAButton}
+        </Links>
+        <Menu isOpen={window.innerWidth > 768} styles={Burger}>
```
Comment (krubenok) [inline src/features/Nav/Navbar.tsx]: Same, could the `theme.screens.smUp` value be used here?

File: src/features/Application/ManageApplicationContainer.tsx [hunk 4]
```diff
@@ -149,53 +149,53 @@ class ManageApplicationContainer extends React.Component<
...
-            validationSchema={getValidationSchema(
-              mode === ManageApplicationModes.CREATE,
-              this.state.pageNumber
-            )}
-          />
+          /> */}
+        <Helmet>
+          <title>
+            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'}
+            Application | McHacks 6
```
[the remark this answers is not in the document]
Comment (krubenok) [reply src/features/Application/ManageApplicationContainer.tsx]: Even if you don’t want to constants file this, this still needs to be fixed. @loreina

**Pull request merged**